general/forall.hpp

```diff
@@ -176,6 +176,34 @@ void CuWrap3D(const int N, DBODY &&d_body,
    MFEM_CUDA_CHECK(cudaGetLastError());
 }
 
+// Macro for launching a 2D tensor kernel
+#define call_2d_ker(KER,NEL,DOFS,QUAD,BZ,NBLOCK,...)         \
```

Comment:
Should these be called `mfem_call_2d_ker` or maybe `mfem_2d_ker` for namespaning?

Reply:
I will integrate them in the MFEM foralls: this can be done later.